pyuvdata/uvdata/uvdata.py

```diff
@@ -2505,9 +3156,20 @@ def phase(
         dec,
         epoch="J2000",
         phase_frame="icrs",
-        use_ant_pos=False,
+        object_name=None,
+        object_type="sidereal",
+        pm_ra=None,
+        pm_dec=None,
+        parallax=None,
+        rad_vel=None,
+        use_ant_pos=False,  # Can we change this to default to True?
```

Comment:
In the context of the new `phase` method, that's still mostly correct - the old _w_-values are used for "unphasing" the data, but when `use_ant_pos=True`, it will only use the antenna positions for calculating the new _uvw_-coordinates. The advantage here is that, in addition to being faster, it doesn't suffer precision penalties from multiple rotations being applied. There will be times where you don't want to do this, like when the antenna positions aren't trustworthy for some reason, but I feel like those instances should be treated as the exception rather than the rule.

Reply:
Discussed this today, and based on that, I will plan on setting this to `True`.